publish-pipeline/publish-core/src/test/resources/test.conf

```diff
@@ -31,7 +31,11 @@ neo4j {
   graph = "domain"
 }
 
-//cloud_storage_type="azure"
+cloud_storage_type="azure"
```

Comment:
@ddevadat Please remove these configuration. circle ci uses env variables to inject actual values. Post removal, circle ci should work.

Reply:
removed